app/views/admin/new_document/index.html.erb

```diff
@@ -0,0 +1,99 @@
+<% content_for :page_title, "New document" %>
+<% content_for :title, "New document" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<%
+  new_document_radio_items = [
+    {
+      value: "call-for-evidence",
+      text: "Call for evidence",
+      bold: true,
+      hint_text: "Use this to request people's views when it is not a consultation.",
+    },
+    {
+      value: "case-study",
+      text: "Case study",
+      bold: true,
+      hint_text: "Use this to share real examples that help users understand a process or an important" +
+        " aspect of government policy covered on GOV.UK.",
+    },
+    {
+      value: "consultation",
+      text: "Consultation",
+      bold: true,
+      hint_text: "Use this for documents requiring a collective agreement across government, and requests for people's view on a question with an outcome.",
+    },
+    {
+      value: "detailed-guide",
+      text: "Detailed guide",
+      bold: true,
+      hint_text: "Use this to tell users the steps they need to take to complete a clearly defined task. They are usually aimed at specialist or professional audiences.",
+    },
+    {
+      value: "document-collection",
+      text: "Document collection",
+      bold: true,
+      hint_text: "Use this to group related documents on a single page for a specific audience or around a specific theme.",
+    },
+    {
+      value: "fatality-notice",
+      text: "Fatality notice",
+      bold: true,
+      hint_text: "Use this to provide official confirmation of the death of a member of the armed forces while on deployment. Ministry of Defence only.",
+    },
+    {
+      value: "news-article",
+      text: "News article",
+      bold: true,
+      hint_text: "Use this for news story, press release, government response, and world news story.",
+    },
+    {
+      value: "publication",
+      text: "Publication",
+      bold: true,
+      hint_text: "Use this for standalone government documents, white papers, strategy documents, and reports.",
+    },
+    {
+      value: "speech",
+      text: "Speech",
+      bold: true,
+      hint_text: "Use this for speeches by ministers or other named spokespeople, and ministerial statements to Parliament.",
+    },
+    {
+      value: "statistical-data-set",
+      text: "Statistical data set",
+      bold: true,
+      hint_text: "Use this for data that you publish monthly or more often without analysis.",
+    },
+  ]
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with url: admin_new_document_options_path do %>
+      <%= render "govuk_publishing_components/components/radio", {
```

Comment:
Where we have a bunch of radio controls we should also have a `<legend>` element inside the `<fieldset>` which is created as part of the `Form radio button ` component. This could be flagged as an accessibility failure/warning. It can be set via the `heading` parameter in the component and should ideally use the page title as its value, removing the page title itself.

Reply:
Accessibility is important - well spotted! Done!

Reply:
Hi @RodneyJohnsonGDS - the `<legend>` change is perfect, ta! Only thing is that we still need the `<% content_for :page_title, "New document" %>` line that has been removed so that the page title appears in the browser tab.

Reply:
Done!